keyboards/reviung39/keymaps/default/keymap.c

```diff
@@ -15,42 +15,44 @@
   ),
   
-  [_LOWER] = REVIUNG39_LAYOUT(\
-    _______,  KC_EXLM,  KC_AT,    KC_HASH,  KC_DLR,    KC_PERC,            KC_CIRC,  KC_AMPR,  KC_ASTR,  KC_LPRN,  KC_RPRN,  KC_DEL,   \
-    _______,  KC_UNDS,  KC_PLUS,  KC_LCBR,  KC_RCBR,   KC_PIPE,            KC_LEFT,  KC_DOWN,  KC_UP,    KC_RGHT,  KC_GRV,   KC_TILD,  \
-    _______,  KC_ESC,   KC_LGUI,  KC_LALT,  KC_CAPS,   KC_DQUO,            KC_HOME,  KC_END,   KC_PGUP,  KC_PGDN,  KC_PSCR,  RSFT_T(KC_SPC),  \
-                                                       _______,  KC_ENT,   _______   \
+  [_LOWER] = LAYOUT_reviung39(
+    _______,  KC_EXLM,  KC_AT,    KC_HASH,  KC_DLR,    KC_PERC,            KC_CIRC,  KC_AMPR,  KC_ASTR,  KC_LPRN,  KC_RPRN,  KC_DEL,
+    _______,  KC_UNDS,  KC_PLUS,  KC_LCBR,  KC_RCBR,   KC_PIPE,            KC_LEFT,  KC_DOWN,  KC_UP,    KC_RGHT,  KC_GRV,   KC_TILD,
+    _______,  KC_ESC,   KC_LGUI,  KC_LALT,  KC_CAPS,   KC_DQUO,            KC_HOME,  KC_END,   KC_PGUP,  KC_PGDN,  KC_PSCR,  RSFT_T(KC_SPC),
+                                                       _______,  KC_ENT,   _______
   ),
   
-  [_RAISE] = REVIUNG39_LAYOUT(\
-    _______,  KC_1,     KC_2,     KC_3,     KC_4,      KC_5,               KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_DEL,   \
-    _______,  KC_MINS,  KC_EQL,   KC_LBRC,  KC_RBRC,   KC_BSLS,            KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    \
-    _______,  KC_ESC,   KC_RGUI,  KC_RALT,  KC_CAPS,   KC_QUOT,            KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   \
-                                                       _______,  KC_BSPC,  _______   \
+  [_RAISE] = LAYOUT_reviung39(
+    _______,  KC_1,     KC_2,     KC_3,     KC_4,      KC_5,               KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_DEL,
+    _______,  KC_MINS,  KC_EQL,   KC_LBRC,  KC_RBRC,   KC_BSLS,            KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,
+    _______,  KC_ESC,   KC_RGUI,  KC_RALT,  KC_CAPS,   KC_QUOT,            KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,
+                                                       _______,  KC_BSPC,  _______
   ),
   
-  [_ADJUST] = REVIUNG39_LAYOUT(\
-    RGB_VAI,   RGB_SAI, RGB_HUI,  RGB_MOD,  XXXXXXX,   RGB_TOG,            KC_EJCT,  XXXXXXX,  XXXXXXX,  KC_BRMU,  KC__VOLUP, KC_VOLU,  \
-    RGB_VAD,   RGB_SAD, RGB_HUD,  RGB_RMOD, XXXXXXX,   XXXXXXX,            XXXXXXX,  XXXXXXX,  XXXXXXX,  KC_BRMD,  KC__VOLDOWN,KC_VOLD, \
-    XXXXXXX,   XXXXXXX, XXXXXXX,  XXXXXXX,  XXXXXXX,   XXXXXXX,            RESET,    XXXXXXX,  XXXXXXX,  XXXXXXX,  KC__MUTE,  KC_MUTE,  \
-                                                       _______,  XXXXXXX,  _______   \
+  [_ADJUST] = LAYOUT_reviung39(
+    RGB_VAI,   RGB_SAI, RGB_HUI,  RGB_MOD,  XXXXXXX,   RGB_TOG,            KC_EJCT,  XXXXXXX,  XXXXXXX,  KC_BRMU,  KC__VOLUP, KC_VOLU,
+    RGB_VAD,   RGB_SAD, RGB_HUD,  RGB_RMOD, XXXXXXX,   XXXXXXX,            XXXXXXX,  XXXXXXX,  XXXXXXX,  KC_BRMD,  KC__VOLDOWN,KC_VOLD,
```

Comment:
No need to use `KC_BRMU/D` and `KC__VOLUP/DOWN`, these are macOS specific, whereas `KC_BRIU/D` and `KC_VOLU/D` will work on all OSes. Same with `KC_MUTE`.

Reply:
thanks

Reply:
Rethinking that volume control is not necessary, I removed it from the default keymap.